roles/gpu_operator_run_gpu-burn/tasks/main.yml

```diff
@@ -43,29 +43,20 @@
 - name: "Let the GPU burn Pods run {{ gpu_burn_time }}seconds"
   command: "sleep {{ gpu_burn_time }}"
 
-- name: Wait for GPU burn Pods to complete
-  with_items: "{{ gpu_burn_gpu_nodes.stdout_lines }}"
-  command:
-    oc get pod/gpu-burn-{{ item }}
-       -n default
-       -o custom-columns=:.status.phase
-       --no-headers
-  register: gpu_burn_gpu_wait
-  until: gpu_burn_gpu_wait.stdout == "Succeeded" or gpu_burn_gpu_wait.stdout == "Error" or gpu_burn_gpu_wait.stdout == "Failed"
-  retries: 10
-  delay: 30
-
 - name: Ensure that the GPU burn ran successfully
   block:
-  - name: Ensure that the GPU burn Pods to complete successfully
+  - name: Wait for GPU burn Pods to complete
     with_items: "{{ gpu_burn_gpu_nodes.stdout_lines }}"
     command:
       oc get pod/gpu-burn-{{ item }}
          -n default
          -o custom-columns=:.status.phase
          --no-headers
-    register: gpu_burn_gpu_test
-    failed_when: gpu_burn_gpu_test.stdout != "Succeeded"
+    register: gpu_burn_gpu_wait
```

Comment:
The naming of this variable seems redundant. I understand gpu_burn_gpu_nodes is the nodes with gpus for running gpu_burn. But why not gpu_burn_wait or gpu_burn_pod_status?